samples/quickstart/model.yaml

```diff
@@ -51,7 +51,7 @@ spec:
               path: /health
               port: 8000
               scheme: HTTP
-            initialDelaySeconds: 120
+            initialDelaySeconds: 240
```

Comment:
This is tricky because the waiting time depends on the model size and network heavily. Maybe quick start with a smaller model makes more sense,  like opt-125m.

Reply:
Yeah reducing the model size is one way to go, it would guarantee quicker startup times. Another way could be increasing the failure threshold, that way we get quick startup times if the network is fast, but don't get pod restarts if it is slow.

Reply:
Should have included machine specs in my original bug ticket. Running on a `g2-standard-4` from GCP, which has a max bandwidth of 10Gb/s, the slowest of the GPU machines, so model download is worst-case.

Reply:
@kerthcet thoughts?

Reply:
https://kubernetes.io/docs/concepts/configuration/liveness-readiness-startup-probes/#startup-probe @jolfr @kerthcet  WDYT?

Reply:
make sense to me

Reply:
@jolfr could you make a change?